### PR TITLE
core: Improve startup time by starting high priority services first

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
This represents a 12% time reduction.
> note: Reduces QGC connection from 102 seconds to 90 seconds total boot time.

### CODE REVIEW

File: core/start-blueos-core [hunk 1]
```diff
@@ -55,18 +55,24 @@ else
     echo "Waring: Using dockers's resolv.conf, this may cause DNS issues if the host's network configuration changes."
 fi
 
+# These services have priority because they do the fundamental for the vehicle to work, and by initializing them first
```
Comment (rafaellehmkuhl) [inline core/start-blueos-core:58]: "Those"

**Review by rafaellehmkuhl — APPROVED** [review]

**Comment by joaoantoniocardoso** [comment]
I'll test a bit more, and see if that sleep is really needed.

**Comment by joaoantoniocardoso** [comment]
Okay, without the 5 seconds sleep the time increases by ~8 seconds, ending with a total net time decrement of 4 seconds only (~4%). So I believe it is worth keeping. 

I also measured the time it takes from starting the priority services to have a connection on QGC and it is about 5 seconds, so 5 seconds of sleep is probably not far from the optimal sleep time.

With that said, I'm opening it again for review.

**Comment by joaoantoniocardoso** [comment]
Oh, just to mention, from that 1min30s, the startup time is actually about 25s, as can be seen in the video below. The CPU profile can be compared with the one from [here](https://github.com/bluerobotics/BlueOS/issues/1808):

https://github.com/bluerobotics/BlueOS/assets/5920286/6816abce-c524-4b98-b63e-21a7eff74efb

**Comment by patrickelectric** [comment]
So.. what's the real gain with this patch ? 4% over 12% ?

**Comment by joaoantoniocardoso** [comment]
> So.. what's the real gain with this patch ? 4% over 12% ?

If we keep the 5 seconds sleep, it's 12%, otherwise, 4%

**Review by patrickelectric — APPROVED** [review]

**Comment by patrickelectric** [comment]
Helps: https://github.com/bluerobotics/BlueOS/issues/1489

**Pull request merged**